src/hct_mis_api/apps/dashboard/serializers.py

```diff
@@ -1,23 +1,30 @@
 from rest_framework import serializers
 
 
-class DashboardHouseholdSerializer(serializers.Serializer):
-    currency = serializers.CharField()
+class DashboardBaseSerializer(serializers.Serializer):
     total_delivered_quantity_usd = serializers.DecimalField(max_digits=12, decimal_places=2)
-    total_delivered_quantity = serializers.DecimalField(max_digits=12, decimal_places=2)
     payments = serializers.IntegerField()
     individuals = serializers.IntegerField()
     households = serializers.IntegerField()
     children_counts = serializers.IntegerField()
-    month = serializers.CharField()
     year = serializers.IntegerField()
     program = serializers.CharField()
     sector = serializers.CharField()
     status = serializers.CharField()
-    admin1 = serializers.CharField()
     fsp = serializers.CharField()
     delivery_types = serializers.CharField()
     pwd_counts = serializers.IntegerField()
     reconciled = serializers.IntegerField()
     finished_payment_plans = serializers.IntegerField()
     total_payment_plans = serializers.IntegerField()
+
+
+class DashboardHouseholdSerializer(DashboardBaseSerializer):
```

Comment:
why don't we have only one serializer?

Reply:
it is ok... we don't need to use all the fields, but we can save 2 class creations

Reply:
Sure. I will use the required argument.

Reply:
no need... aren't these just to read info?